srml/staking/src/lib.rs

```diff
@@ -994,16 +1010,16 @@ impl<T: Trait> Module<T> {
 			}
 			safe_mul_rational(exposure.own)
 		};
+
 		imbalance.maybe_subsume(Self::make_payout(stash, validator_cut + off_the_table));
-		T::Reward::on_unbalanced(imbalance);
+
+		imbalance
 	}
 
 	/// Session has just ended. Provide the validator set for the next session if it's an era-end.
 	fn new_session(session_index: SessionIndex) -> Option<Vec<T::AccountId>> {
-		// accumulate good session reward
-		let reward = Self::current_session_reward();
-		<CurrentEraReward<T>>::mutate(|r| *r += reward);
-
+		// TODO TODO: previous implementation was accumulating reward here, should we accumulate
+		// duration in a way ?
```

Comment:
The purpose of this accumulation was that in the past the session length was variable (based on network performance) and the reward should have changed proportionally. and I __think__ now it is fixed so there should be no need for it. Thou' I am not sure if this has landed already.